sdks/python/apache_beam/utils/retry.py

```diff
@@ -176,7 +176,7 @@ def wrapper(*args, **kwargs):
               sleep_interval = retry_intervals.next()
             except StopIteration:
               # Re-raise the original exception since we finished the retries.
-              raise exn, None, exn_traceback
+              raise exn, None, exn_traceback  # pylint: disable=raising-bad-type
```

Comment:
What is this? Maybe we should disable some of these lint warnings?

Do we need this or `too-many-nested-blocks`?